charts/camunda-platform/templates/connectors/deployment.yaml

```diff
@@ -85,11 +85,14 @@ spec:
           {{- if .Values.connectors.env}}
             {{ .Values.connectors.env | toYaml | nindent 12 }}
           {{- end }}
-          {{- if .Values.global.identity.auth.enabled }}
           envFrom:
+          {{- if .Values.global.identity.auth.enabled }}
             - configMapRef:
                 name: {{ include "camundaPlatform.fullname" . }}-identity-env-vars
           {{- end }}
+          {{- if .Values.connectors.envFrom }}
```

Comment:
It should be `{{- if .Values.connectors.envFrom -}}` (with minus at the end) to remove the extra line before the if.

Reply:
The ones that needed this were the ones that had the preconfigured `envFrom` the others I did not add.